lib/slack-notifier/util/http_client.rb

```diff
@@ -21,7 +23,14 @@ def initialize uri, params
         end
 
         def call
-          http_obj.request request_obj
+          response = http_obj.request(request_obj)
```

Comment:
you think it'd be too "clever"  to use `tap` and skip the assignment?

like:
```ruby
def call
  http_obj.request(request_obj).tap do |response|
    unless response.is_a? Net::HTTPSuccess
      raise Slack::Notifier::APIError, #...
    end
  end
end
```

(I'm writing php now at the day job so I may be over-rubying it 😉)